packages/derived-wallet-solana/src/SolanaDerivedWallet.ts

```diff
@@ -203,14 +203,14 @@ export class SolanaDerivedWallet implements AptosWallet {
   // region Signatures
 
   async signMessage(input: AptosSignMessageInput): Promise<UserResponse<AptosSignMessageOutput>> {
-    const chainId = input.chainId ? this.defaultNetwork === Network.DEVNET ? await fetchDevnetChainId() : NetworkToChainId[this.defaultNetwork] : undefined;
     return signAptosMessageWithSolana({
       solanaWallet: this.solanaWallet,
       authenticationFunction: this.authenticationFunction,
       messageInput: {
         ...input,
-        chainId,
+        chainId:  NetworkToChainId[this.defaultNetwork],
```

Comment:
```suggestion
        chainId: NetworkToChainId[this.defaultNetwork],
```

we should probably set up prettier / eslint at some point